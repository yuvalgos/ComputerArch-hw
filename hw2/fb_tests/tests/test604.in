r 0x7c3fc
r 0x7c3d8
r 0x7c3e0
r 0x7c404
r 0x7c434
w 0x7c3c4
w 0x7c3f8
w 0x7c3bc
w 0x7c3d8
r 0x7c428
r 0x7c404
w 0x7c3c0
r 0x7c404
r 0x7c3f0
w 0x7c41c
w 0x7c3c0
r 0x7c3dc
w 0x7c420
r 0x7c42c
r 0x7c440
r 0x7c448
r 0x7c3ec
r 0x7c458
w 0x7c424
w 0x7c3cc
r 0x7c444
w 0x7c440
w 0x7c444
r 0x7c454
r 0x7c420
r 0x7c3f0
r 0x7c3d4
r 0x7c414
w 0x7c3e0
w 0x7c40c
w 0x7c454
r 0x7c404
r 0x7c418
w 0x7c3e8
w 0x7c3e0
r 0x7c448
