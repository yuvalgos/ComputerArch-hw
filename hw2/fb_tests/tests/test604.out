L1miss=0.829 L2miss=0.647 AccTimeAvg=141.976
